pkg/dpll-netlink/dpll-uapi.go

```diff
@@ -3,9 +3,14 @@
 
 package dpll_netlink
 
-import "fmt"
+import (
+	"encoding/json"
+	"fmt"
+)
 
 const DPLL_MCGRP_MONITOR = "monitor"
+const DPLL_PHASE_OFFSET_DIVIDER = 1000
```

Comment:
where is this used ?DPLL_PHASE_OFFSET_DIVIDER

Reply:
d.phaseOffset = int64(math.Round(float64(phaseOffset / nl.DPLL_PHASE_OFFSET_DIVIDER / 1000)))
this is from pkg/dpll/dpll.go